content/docs/concepts/quality_gates/index.md

```diff
@@ -5,11 +5,11 @@ weight: 2
 keywords: [keptn, use-cases]
 ---
 
-> When developing a software product, sooner or later you need to update a service in a production environment. To conduct this in a controlled manner and without impacting end-user experience, the quality of the new service has to be ensured in an automated way. 
+> When developing a software product, sooner than later you need to deliver a new deployment of your service in a production environment. To conduct this in a controlled manner and without impacting end-user experience, the quality of the new service has to be ensured in an automated way. 
```

Comment:
I would suggest to say "deliver a new version of your service". You usually don't deliver a deployment.

Reply:
changed it :)